main.tf

```diff
@@ -141,27 +141,33 @@ module "eks" {
   }
 
   ################################################################################
-  # Handle BYO IAM policy
+  # Handle BYO IAM Roles & Policies
   ################################################################################
-  create_iam_role                                = var.cluster_iam_role_name == null ? true : false   # v17: manage_cluster_iam_resources
-  iam_role_name                                  = var.cluster_iam_role_name                          # v17: cluster_iam_role_name
+  # BYO - EKS Cluster IAM Role
+  create_iam_role = var.cluster_iam_role_arn == null ? true : false # v17: manage_cluster_iam_resources
+  iam_role_arn    = var.cluster_iam_role_arn                        # v17: cluster_iam_role_arn
+
   iam_role_additional_policies = [
     "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
   ]
 
   ## Use this to define any values that are common and applicable to all Node Groups 
   eks_managed_node_group_defaults = {
-    create_security_group   = false
-    vpc_security_group_ids  = [local.workers_security_group_id]
+    create_security_group  = false
+    vpc_security_group_ids = [local.workers_security_group_id]
+
+    # BYO - EKS Workers IAM Role
+    create_iam_role = var.workers_iam_role_arn == null ? true : false # v17: manage_workers_iam_resources
+    iam_role_arn    = var.workers_iam_role_arn                        # v17: worker_iam_role_arn
```

Comment:
Remove v17 comment elements.

Reply:
Removed additional comments for the workers IAM role.